app/controllers/users_controller.rb

```diff
@@ -1,4 +1,36 @@
 class UsersController < ApplicationController
+  before_action :load_user, only: %i(show edit update)
+  def edit; end
+
+  def show; end
+  
+  def index
+    render html: @users = User.all
```

Comment:
What is the purpose of using this?

Reply:
I will remove right now :))
Before make pull request , i was pull branh "in progress" other member